### PR TITLE
Correctly response error

### PR DESCRIPTION
**Pull request by minsulee2**
Now error is correctly sent and written in the blockchain

### CODE REVIEW

File: handlers/openai_ainize_handlers.js [hunk 1]
```diff
@@ -64,14 +64,18 @@ class OpenaiAinizeHandler {
...
+        throw ErrorUtil.setCustomError(response.status, response.message);
+      }
 
       // FIXME(minsu): this is tempolar approach.
       if (!REST_MODE) await ainizeAdmin.internal.handleRequest(req, 0, AINIZE_STATUS.SUCCESS, response.data);
-      // FIXME(minsu): response can be either ok or err.
       res.status(200).json(Utils.serializeMessage(`${jobType} ok`, response.data));
     } catch (error) {
-      throw ErrorUtil.setCustomError(500, error);
+      next(ErrorUtil.setCustomError(error.status, error.message));
```
Comment (jiyoung-an) [inline handlers/openai_ainize_handlers.js:78]: 👍

File: handlers/utils.js [hunk 2]
```diff
@@ -6,8 +6,6 @@ class Utils {
   static TrimUnnecessaryDataForResponseData(data) {
     delete data.tools;
     delete data.file_ids;
-    // FIXME(minsu): metadata is necessary to show. but need to discuss how to cope with {}.
-    delete data.metadata;
```
Comment (jiyoung-an) [inline handlers/utils.js:10]: 👍

**Review by jiyoung-an — APPROVED** [review]
LGTM, Thanks!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by minsulee2** [comment]
Thanks for the fast reviews! Merging now

**Pull request merged**